ansible/setup-partyhall.yaml

```diff
@@ -6,13 +6,13 @@
     - name: 'Install partyhall dependencies'
       become: yes
       apt:
-        name: [ 'libvips' ]
+        name: [ 'libvips', 'pulseaudio', 'pavucontrol' ]
```

Comment:
No, no pulseaudio, we're using pipewire, and its done in the global thing anyway